lookout/style/typos/corrector.py

```diff
@@ -213,33 +203,26 @@ def suggest_by_batches(self, data: pandas.DataFrame, n_candidates: int = 3,
         :param n_candidates: Number of most probable candidates to return.
         :param return_all: False to return suggestions only for corrected tokens.
         :param batch_size: Batch size.
-        :param start_pool_size: Length of data, starting from which multiprocessing is desired.
```

Comment:
I hope that all parameters description are moved somewhere and not just deleted.

Reply:
Yes that is so